stock_prices/stock_prices.py

```diff
@@ -3,7 +3,68 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+    # max_profit = -1000
+    # min_value = prices[0]
+    # max_value = -1000
+    # if len(prices) < 2:
+    #     return 0
+    # for price in prices:
+    #     if price < min_value:
+    #         min_value = price
+    #         # print(min_value)
+    #         min_index = prices.index(min_value)
+    #         # print(min_index)
+    # for price in prices[min_index:]:
+    #     print(min_value)
+    #     print(prices[min_index:])
+    #     if price >= max_value:
+    #         max_value = price
+    #         print(max_value)
+    #     if max_value == prices[-1]:
+    #         return prices[-1] * -1
+    # return(max_value - min_value)
+
+    #Find max profit from the highest 
+    # highest then look for highest in right
+
+    # find max profit from lowest
+    # lowest then look right for highest number
+
+    # return higher of each
+
+
+
+    # if len(prices) < 2:
+    #     return 0
+    # for price in prices:
+    #     current_price = price
+    #     # if current_price + 1 < len(prices):
+    #     max_index = prices.index(current_price)
+    #     max_price = prices[max_index]
+    #     print("CUR", current_price)
+    #     if max_price - current_price > max_profit:
+    #         max_profit = max_price - current_price
+    # return max_profit
+    max_profit = 0
+    for i in range(0, len(prices)):
+        current_price = prices[i]
+        # print('current PRICE', current_price)
+        for j in range(i, len(prices)):
+            next_price = prices[j]
+            # print('next PRICE', next_price)
+            current_profit = next_price - current_price
+            # print('CURRENT PROFIT', current_profit)
+
+            if current_profit > max_profit:
+                max_profit = current_profit
+                # print('MAX PROFIT', max_profit)
+    if max_profit == 0:
+        return prices[-1] * -1
```

Comment:
I thought this line was clever. I wouldn't have thought to multiply by -1. when I did this I just had the return be -(prices[-1])